Add dateonly and timeonly support to pickers. (#957)
telerik/maui-docs: controls/datepicker/dateonly.md

---

> title: Date Only Support

Reviewer: In this case, I suggest keeping the original naming as "Date Only" doesn't appear to be a common term and it makes sense only when used as a name for the struct.
```suggestion
title: Using DateOnly
```

page_title: .NET MAUI Date Picker Documentation - Date Only Support
description: Set date only type in the Telerik UI for .NET MAUI DatePicker.
position: 4
slug: datepicker-dateonly
---

> # Date Only Support in .NET MAUI DatePicker

Reviewer: ```suggestion
# Using DateOnly in the .NET MAUI DatePicker
```

> The DatePicker has a support for `DateOnly`. You can set a `DateOnly` value to the `MinimumDate`, `MaximumDate`, `Date` and `DefaultHighlightedDate` properties by using the Telerik `DateOnlyToDateTimeConverter `converter. The converter converts `System.DateOnly` to `System.DateTime` and converts back `System.DateTime` to `System.DateOnly`.

Reviewer: - Describing a simple potential use case allows you to set the context for the article that describes a specific feature.
- This comment also applies to the rest of the articles in this PR.

```suggestion
If your application needs to store or manipulate only the dates without times, you can take advantage of the `DateOnly` struct when working with the DatePicker control.

You can set a `DateOnly` value to the `MinimumDate`, `MaximumDate`, `Date`, and `DefaultHighlightedDate` properties by using the Telerik `DateOnlyToDateTimeConverter `converter. The converter converts `System.DateOnly` to `System.DateTime` and `System.DateTime` to `System.DateOnly`.
```
